libr/debug/p/debug_native.c

```diff
@@ -505,9 +505,11 @@ static RDebugReasonType r_debug_native_wait (RDebug *dbg, int pid) {
 				return R_DEBUG_REASON_ERROR;
 			}
 			reason = dbg->reason.type;
+#ifdef WIFCONTINUED
```

Comment:
Apparently NetBSD actually have WIFCONTINUED (http://netbsd.gw.com/cgi-bin/man-cgi?wait+2+NetBSD-current). Maybe we are missing an header?